cdp-docs/docs/project_resources/sql_style_guide.md

```diff
@@ -341,7 +371,9 @@ with paying_customers as (
 select ...
 from paying_customers_per_month
 ```
-
+| **SQLFluff Rule Code** | **SQLFluff Fix compatible** |
+|:---|:---|
+| [LT07](https://docs.sqlfluff.com/en/stable/rules.html#rule-LT07), <br /> [LT08](https://docs.sqlfluff.com/en/stable/rules.html#rule-LT08) | Yes |
```

Comment:
Is the `<br />` intentional? Wondering bc it doesn't match the format of the other listed rule codes.

Reply:
Hi Insuh -- It was intentional when I thought I was keeping one rule code per line, I missed this one.  Thank you!